frontend/entrypoint.sh

```diff
@@ -0,0 +1,30 @@
+#! /bin/sh
```

Comment:
Musst die file entrypoint.sh heißen? Wenn ja, resolve this.

Wenn nein, sollen wir's dann irgendwie nenen das man weiss um was es geht?